tests/portforward_test.go

```diff
@@ -49,19 +46,16 @@ var _ = Describe("[sig-compute]PortForward", decorators.SigCompute, func() {
 		virtClient = kubevirt.Client()
 	})
 
-	It("should successfully open connection to guest", func() {
+	It("should successfully open connection to guest", decorators.Conformance, func() {
 		vmi := libvmifact.NewFedora(
 			libnet.WithMasqueradeNetworking(),
-			libvmi.WithNamespace(testsuite.NamespaceTestDefault),
 		)
-		libvmops.RunVMIAndExpectLaunchIgnoreWarnings(vmi, 180)
+		vmi = libvmops.RunVMIAndExpectLaunchIgnoreWarnings(vmi, 180)
```

Comment:
@xpivarc Previously the vmi was created with `libvmi.WithNamespace(testsuite.NamespaceTestDefault)`, and when we later perform the portforward in the `vmi.Namespace` ns it was working.
Having dropped the  `libvmi.WithNamespace(testsuite.NamespaceTestDefault)`, we need to update our local vmi object with the one retrieved during launch.
@0xFelix FYI

Reply:
Nice :)